src/st/st-scroll-view.c

```diff
@@ -550,8 +551,8 @@ st_scroll_view_get_preferred_height (ClutterActor *actor,
                                      gfloat       *natural_height_p)
 {
   StScrollViewPrivate *priv = ST_SCROLL_VIEW (actor)->priv;
-  StThemeNode *theme_node = st_widget_get_theme_node (ST_WIDGET (actor));
-  gboolean account_for_hscrollbar = FALSE;
+  StThemeNode *theme_node;
+  gboolean account_for_hscrollbar;
```

Comment:
This will be initialized as `0` implicitly, which is the `FALSE` value. I think explicit declaration here is cleaner.

Reply:
OK